doc/modules/svm.rst

```diff
@@ -677,7 +677,7 @@ The primal problem can be equivalently formulated as
 
 .. math::
 
-    \min_ {w, b} \frac{1}{2} w^T w + C \sum_{i=1}\max(0, y_i (w^T \phi(x_i) + b)),
+    \min_ {w, b} \frac{1}{2} w^T w + C \sum_{i=1}\max(0, 1-y_i (w^T \phi(x_i) + b)),
```

Comment:
The `-` should include the bias term as well:
(I'm not sure this will render nicely in latex, we should double check:)

```suggestion
    \min_ {w, b} \frac{1}{2} w^T w + C \sum_{i=1}\max(0, 1 - [ y_i (w^T \phi(x_i) + b] )),
```

Reply:
I think square brackets are not so relevant in this case, but I'll pr with this change.